ingest/ingest_pipeline.py

```diff
@@ -232,8 +234,8 @@ def load_subsample(self, doc):
 
     def has_valid_metadata_convention(self):
         """ Determines if cell metadata file follows metadata convention"""
-        with open(self.JSON_CONVENTION, 'r') as f:
-            convention = json.load(f)
+        json_file = IngestFiles(self.JSON_CONVENTION, ['application/json'])
```

Comment:
just my ignorance here, should we need to close this filehandle somewhere? In my tests for validate_metadata, I'd get errors about filehandles. I'm trying to understand best practice around this.

Reply:
0dba75b.